src/Avalonia.Controls/Notifications/WindowNotificationManager.cs

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) The Avalonia Project. All rights reserved.
+// Licensed under the MIT license. See licence.md file in the project root for full license information.
+
+using System;
+using System.Collections;
+using System.Linq;
+using System.Reactive.Linq;
+using System.Threading.Tasks;
+using Avalonia.Controls.Primitives;
+using Avalonia.VisualTree;
+
+namespace Avalonia.Controls.Notifications
+{
+    /// <summary>
+    /// Defines a WindowNotificationManager control.
+    /// </summary>
+    public class WindowNotificationManager : TemplatedControl, IManagedNotificationManager
+    {
+        private IList _items;
+
+        /// <summary>
+        /// Defines the <see cref="Position"/> property.
+        /// </summary>
+        public static readonly StyledProperty<NotificationPosition> PositionProperty =
+          AvaloniaProperty.Register<WindowNotificationManager, NotificationPosition>(nameof(Position), NotificationPosition.TopRight);
+
+        /// <summary>
+        /// Defines which corner of the screen notifications can be displayed in. <seealso cref="NotificationPosition"/>.
```

Comment:
I don't think `<seealso>` tags can appear in `<summary>` tags? I think it should come after the summary (I could be mistaken).